examples/titanic/README.md

```diff
@@ -15,19 +15,19 @@ In order to run this example, you'll need to:
 * [pull the `substra-tools` docker images](https://github.com/substrafoundation/substra-tools#pull-from-private-docker-registry)
 * create a substra profile to define the substra network to target, for instance:
     ```sh
-    substra config --profile node-1 --username node-1 --password 'p@$swr0d44' http://substra-backend.node-1.com
-    substra login --profile node-1
+    substra config --profile node-1 http://substra-backend.node-1.com
+    substra login --profile node-1 --username node-1 --password 'p@$swr0d44'
     ```
 
 ## Data preparation
 
-The first step will be to generate train and test data samples from the 
-[Kaggle challenge source](https://www.kaggle.com/c/titanic/data). The raw `train.csv` file is available in the `data` 
+The first step will be to generate train and test data samples from the
```

Comment:
To avoid adding a lot of unwanted changes (done by your editor I guess), you could use the `git add -p` to select only the desired changes.

Reply:
There is also an option on github to not display these changes. Forgot my previous message.